File: 1460-make-two-arrays-equal-by-reversing-subarrays/1460-make-two-arrays-equal-by-reversing-subarrays.py
class Solution(object):
    def canBeEqual(self, target, arr):
        """
        :type target: List[int]
        :type arr: List[int]
        :rtype: bool
        """
        target.sort()
        arr.sort()
        for i in range(0,len(target)):
            if target[i]!=arr[i]:
                return False
        return True

File: 1460-make-two-arrays-equal-by-reversing-subarrays/1460-make-two-arrays-equal-by-reversing-subarrays.js
/**
 * @param {number[]} target
 * @param {number[]} arr
 * @return {boolean}
 */
var canBeEqual = function(target, arr) {
    target.sort((a, b) => a - b);
    let a = arr.sort((a,b)=>a-b);
    let i = 0;
    let j = 0;
    while (i <a.length && j<arr.length){
        if(target[i]!==a[j]){
            return false
        }
        i++
        j++
    }
    return true
};